src/index.html

```diff
@@ -11,6 +11,50 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form"
+      action="#"
+      method="get"
+      data-qa="big"
+    >
+
+      <label class="firstSearch">
```

Comment:
better use kebab-case for classnames

Reply:
I tried use a camel-case, unless kebab-case and camel-case are diferent. It seems to me these possibless to call class are same way.